pkg/restapi/task.go

```diff
@@ -232,6 +233,11 @@ func (h *taskHandler) getTarget(w http.ResponseWriter, r *http.Request) {
 		}
 	case scheduler.RepairTask:
 		if t, err = h.Repair.GetTarget(r.Context(), newTask.ClusterID, p); err != nil {
+			if errors.Is(err, scyllaclient.ErrHostInvalidResponse) {
+				respondError(w, r, err, "One of the nodes is down. Task cannot be created.\n"+
+					"Check --ignore-down-hosts flag if you wish to exclude the node from the repair.")
+				return
+			}
```

Comment:
It's rather difficult to maintain this approach when the same error message is present at 3 separate places.

Also, this approach requires ifing the error in 3 places now:
- in the client function
- in create/update/dry-run of a task
- in the CLI

Isn't it more complicated than just returning human readable error message from `GetTarget` directly?

Reply:
Although there isn't really any ifing out in the CLI itself.

Reply:
But this message is returned from the API call directly.
I need to if here, because I must find what's the root cause of an error and what to describe to the end user.